### DISCUSSION TITLE
Facing issue while running mp4 videos in application

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Ed/Community#656] [General Discussion]
I have generated the build of release 5.0 with sdk 33.
I can consume all the other content format leaving mp4 format videos.
Any idea about the issue attaching the screenshot:
![image](https://github.com/Sunbird-Ed/Community/assets/72926048/8bc331b4-6a00-44d4-ab85-0f79a31cba03)

**Comment by siddhishinde0723** [comment]
@swayangjit any one who can help me with this.

**Reply by swayangjit** [reply]
@siddhishinde0723  We are looking into this will update you once we are done with our analysis

**Reply by swayangjit** [reply]
@siddhishinde0723  Can you please add the content metadata

**Reply by siddhishinde0723** [reply]
@swayangjit Did you got any solution for this

**Comment by swayangjit** [comment]
@siddhishinde0723 can you please try giving following permissions and try

Images and photos | READ_MEDIA_IMAGES
-- | --
Videos | READ_MEDIA_VIDEO
Audio files | READ_MEDIA_AUDIO

Images and photos	[READ_MEDIA_IMAGES](https://developer.android.com/reference/android/Manifest.permission#READ_MEDIA_IMAGES)
Videos	[READ_MEDIA_VIDEO](https://developer.android.com/reference/android/Manifest.permission#READ_MEDIA_VIDEO)
Audio files	[READ_MEDIA_AUDIO](https://developer.android.com/reference/android/Manifest.permission#READ_MEDIA_AUDIO)

**Reply by siddhishinde0723** [reply]
@swayangjit I am facing an issue with mp4. even after giving permission.
